models/data_maps.go

```diff
@@ -29,6 +31,20 @@ type DataMap struct {
 	Address     string    `json:"address" db:"address"`
 }
 
+func init() {
+	SetChunkStatuses()
+}
+
+func SetChunkStatuses() {
```

Comment:
We can use native golang enums like this: https://github.com/oysterprotocol/brokernode/blob/da5d641488ecda2552fe03ed917c701db31a9487/models/upload_sessions.go#L13

So we have compiler type checks instead of strings

Reply:
I would also start from int 1 instead of 0 since golang treats 0 as an empty value, we might get unexpected behavior

Reply:
Cool, I'll change this.